python/src/otel/tests/test_otel.py

```diff
@@ -241,77 +261,16 @@ def test_parent_context_from_lambda_event(self):
                 # NOT Active Tracing
                 _X_AMZN_TRACE_ID: MOCK_XRAY_TRACE_CONTEXT_NOT_SAMPLED,
                 # NOT using the X-Ray Propagator
-                "OTEL_PROPAGATORS": "tracecontext",
+                OTEL_PROPAGATORS: "tracecontext",
             },
         )
         test_env_patch.start()
 
         mock_execute_lambda(
             {
                 "headers": {
-                    "traceparent": MOCK_W3C_TRACE_CONTEXT_SAMPLED,
-                    "tracestate": f"{MOCK_W3C_TRACE_STATE_KEY}={MOCK_W3C_TRACE_STATE_VALUE},foo=1,bar=2",
-                }
-            }
-        )
-
-        spans = self.memory_exporter.get_finished_spans()
-
-        assert spans
-
-        self.assertEqual(len(spans), 1)
-        span = spans[0]
-        self.assertEqual(span.get_span_context().trace_id, MOCK_W3C_TRACE_ID)
-
-        parent_context = span.parent
-        self.assertEqual(
-            parent_context.trace_id, span.get_span_context().trace_id
-        )
-        self.assertEqual(parent_context.span_id, MOCK_W3C_PARENT_SPAN_ID)
-        self.assertEqual(len(parent_context.trace_state), 3)
-        self.assertEqual(
-            parent_context.trace_state.get(MOCK_W3C_TRACE_STATE_KEY),
-            MOCK_W3C_TRACE_STATE_VALUE,
-        )
-        self.assertTrue(parent_context.is_remote)
-
-        test_env_patch.stop()
-
-    def test_using_custom_extractor(self):
```

Comment:
This test [is already upstream](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/a9507b3d1daf79b412b67fd97ddb4e2b7117ebcd/instrumentation/opentelemetry-instrumentation-aws-lambda/tests/test_aws_lambda_instrumentation_manual.py#L197) so we don't need to duplicate it here.